### PR TITLE
[Enhancement] Updating stub profile and adding makefile

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @abhi18av as part of refining my work I'll start to make small enhancements and here goes my first one: Updating the stub profile and adding a Makefile for a easy testing. And here goes my changes.

## PR contents:
* makefile
* mock data generator updates
* stub profile updates

As always feel free to make to criticize or suggest any changes, I'm available to discuss everything with you!
Hope we have a merge!

### CODE REVIEW

File: data/mock_data/generate-mock-files.sh [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+touch ./data/mock_data/AC03_R1.fastq.gz
+touch ./data/mock_data/AC03_R2.fastq.gz
+touch ./data/mock_data/AD04_R1.fastq.gz
+touch ./data/mock_data/AD05_R2.fastq.gz
+touch ./data/mock_data/AE06_R1.fastq.gz
+touch ./data/mock_data/AE06_R2.fastq.gz
+touch ./data/mock_data/AD07_R1.fastq.gz
+touch ./data/mock_data/AD07_R2.fastq.gz
+touch ./data/mock_data/GenomeAnalysisTK.jar
+touch ./data/mock_data/NC000124.fasta
```
Comment (abhi18av) [inline data/mock_data/generate-mock-files.sh]: @Mxrcon , just a small comment here, let's use the MTB reference fasta ID as default :)

Comment (Mxrcon) [reply data/mock_data/generate-mock-files.sh]: Yes sure!

**Comment by Mxrcon** [comment]
@abhi18av I've fixed the reference name, so if it's okay to you, I'll merge this pr!

**Review by abhi18av — APPROVED** [review]
Yup, looks good to merge - please go ahead :)

**Pull request merged**